agent/src/agent/cli/pipeline.py

```diff
@@ -339,14 +340,31 @@ def export(dir_path):
     """
     Export pipelines' config into file
     """
+
+    def _export_file(path, pipeline_name):
```

Comment:
the file might not exist anymore inside the container (for example if the container is recreated) you need to copy the content of the file from the config that is stored in the dataabse